.github/workflows/content-release.yml

```diff
@@ -5,19 +5,18 @@ permissions:
   contents: read  # This is required for actions/checkout
 
 on:
+  # Runs on API call. Used for CMS-driven build triggers.
   repository_dispatch:
     types: [content-release]
-  workflow_run:
```

Comment:
This trigger off production tag is deliberately removed here. We add it to the dev and staging builds.